.github/workflows/w3c-integration-test.yml

```diff
@@ -0,0 +1,20 @@
+name: Run w3c tests on push
+
+on: [push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+
+      - name: Install and Build 🔧
+        run: |
+          npm install --ignore-scripts
+          npx lerna bootstrap --scope={propagation-tests,@opentelemetry-api} --include-dependencies
+
+      - name: Run W3C Test harness
+        run: |
+          chmod 755 ./scripts/tracecontext-integration-test.sh
```

Comment:
Should not have to chmod in the test. Executable bit is tracked by git. If you make your local executable, you should be able to check that in as a change.

Reply:
cool, thanks, will change.